src/constants/extensions.ts

```diff
@@ -56,8 +65,7 @@ const oneTransactionPaymentMessages = {
   },
   oneTxPaymentDescriptionLong: {
     id: `${oneTransactionPaymentName}.descriptionLong`,
-    defaultMessage:
-      'Make quick and simple payments to members or any address on the same network.',
+    defaultMessage: `<p>The One Transaction Payment Extension allows payments to a single account at a time with one type of token.</p><p>Note: This extension is installed and enabled by default and can not be removed.</p><h4>How the extension works</h4><ul><li>A DAO creates a payment Action (or Motion, if enabled) to make a payment to a selected recipient, e.g. “Pay Alice 100 xDAI”.</li><li>You can pay any member or address any availble funds from the Colony.</li><li>You are also able to select the specific team funds will be paid from.</li></ul><h4>Useful for:</h4><ul><li>Paying contributors to the DAO.</li><li>Paying for services, or transferring funds externally.</li></ul>`,
```

Comment:
Really not happy with having tags inside the translation string. 

Yes, I know we parse them out, but this will become a problem, when we'll have more languages other than English, as it'll make paragraph splitting complicated

Reply:
Yeah, I agree. I guess one way would be to split this up or even use a markdown parser (is that a thing?) here.

Reply:
we don't have a markdown parser here, I'd vote for using markdown but maybe not in the scope of this PR :D

Reply:
Sure. Then I guess leave a todo comment here.